public/video-ui/src/components/VideoEdit/ContentFlags.js

```diff
@@ -0,0 +1,47 @@
+import React, {PropTypes} from 'react';
+
+export default class ContentFlags extends React.Component {
+
+  renderLegallySensitiveCheckbox() {
+    return (
+      <div className="form-checkbox" >
+        <input
+          id="legallySensitive"
+          className="form-checkbox__input"
+          type="checkbox"
+          checked={this.props.video.legallySensitive}
+          onChange={this.updateFlag.bind(this)}
+        />
+        <label htmlFor="legallySensitive" className="form-checkbox__toggle"></label>
+      </div>
+    )
+  }
+
+  updateFlag(e) {
+    e.preventDefault();
+    const flag = {[e.target.id]: e.target.checked}
+    const video = Object.assign({}, this.props.video, flag);
+    this.props.saveAndUpdateVideo(video)
+  }
+
+  render(){
+    return(
+      <div>
+      <p className="details-list__title">Filters</p>
+        <table className="details-list">
```

Comment:
It seemed sensible when there was going to be lots of switches...Simplified the code